sdk/tables/azure-data-tables/src/tables_clients.cpp

```diff
@@ -407,6 +418,20 @@ TableClient::TableClient(
       std::move(perOperationPolicies2));
 }
 
+TableClient::TableClient(
+    const std::string& serviceUrl,
+    std::shared_ptr<Azure::Data::Tables::Credentials::SharedKeyCredential> credential,
+    Azure::Data::Tables::Sas::TablesSasBuilder sasBuilder,
```

Comment:
Why does this take the sasBuilder by value but the other constructor takes it by reference?

It's also feels weird that you take an SasBuilder *and* a SharedKeyCredential.  Looking at the storage clients, they don't seem to follow the same pattern.

Why is Tables different?